lib/schedule/csv.ex

```diff
@@ -45,8 +45,7 @@ defmodule Schedule.Csv do
     parser = Keyword.get(options, :parse, & &1)
 
     file_binary
-    |> String.split("\n")
-    |> Enum.reject(&(&1 == ""))
+    |> String.codepoints()
```

Comment:
This was the breaking change. It seems like in the 2.x -> 3.x upgrade, the CSV library started to treat lists as though they should be concatenated, rather than as separate lines. 

It now expects a stream of characters, and explicitly requires the newlines to be present.

Reply:
I'm a little surprised by `codepoints` being the answer now [there's no other way to turn a bitstring into a String? is this the best way to "parse" UTF-8 on the fly?]

But I'm so glad to have this solved!

Reply:
I did just try this with [`String.chunk(:valid)/2`](https://hexdocs.pm/elixir/String.html#chunk/2) and it worked for me, but I'm not sure that the behavior of `chunk` would be desired.

Reply:
Actually I wonder if something like 👇 would work

```elixir
[file_binary]
|> CSV.decode!(...)
# ...
```

Reply:
[I tried it and it does seem to work](https://github.com/mbta/skate/pull/2299/commits/eac20e802a87570b599a41b79a3b9aed9d248a55), but I'm not sure whether it's better. What do you think?

Reply:
It's certainly unfamiliar syntax, but this seems fine in Elixir?
IMO it's better for perf, because then (it seems) only one "concat" would happen? instead of concatenating every codepoint.